### ISSUE TITLE
Exception with cascade if HTML file is in nested folder

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#2093]
@childsb 

When testing Preview501, I played with cascade logic in combination with nested folders, and ran into an exception. Steps to reproduce:
- New HTML File - hi-fi page - but put it into a subfolder (eg folder1/)
- Add a Dijit Button
- Open up cascade for 'width'
- Type 'aaa' into class name for the Button - cascade list now shows a proposed new style rule for class 'aaa'
- Click-select on the radio button for the proposed new style rule

Result=> Exception with getResource call within Cascade.js. Here is minified source code where exception occurred. The cssFile call is running null.

```
},_onChangeRemove:function(_5ec){
var _5ed=_5ec.target;
this._changeValue(_5ed,null);
this._updateCascadeList();
},_onChange:function(_5ee){
var loc=null;
if(this._values[_5ee.target].type=="element.style"){
loc=this._getBaseLocation();
}else{
if(this._values[_5ee.target].type=="proposal"){
var _5ef=this.context.getModel();
var _5f0=_5ef.find({elementType:"CSSFile",relativeURL:this._values[_5ee.target].targetFile},true);
loc=_5f0.getResource();
```

**Comment by JonFerraiolo** [comment]
Reopening. I committed a fix to branch AppCssNestedHtml with de1b648a693957a42f468885111bd1ae557d6aa8, but haven't yet merged into master until investigating what's going on with mangling of the app.css file.

**Comment by JonFerraiolo** [comment]
I convinced myself that my fixes didn't regress anything, so checked in and closed.
